features/specs/gezag-minderjarige/3.2-indicatie-gezag-minderjarige.feature

```diff
@@ -211,17 +197,13 @@ Functionaliteit: 3.2 - Achterhalen gezag na uitspraak
     Dan heeft de response een persoon met de volgende gegevens
       | naam                | waarde    |
       | burgerservicenummer | 000000091 |
-#      En heeft de persoon een 'gezag' met de volgende gegevens
-#      | naam                             | waarde                    |
-#      | type                             | GezamenlijkGezag          |
-#      | minderjarige.burgerservicenummer | 000000091                 |
-#      | ouder.burgerservicenummer        | 000000061                 |
-#      | derde.burgerservicenummer        | 000000085                 |
     En heeft de persoon een 'gezag' met de volgende gegevens
-      | naam                             | waarde    |
-      | type                             | Voogdij   |
-      | minderjarige.burgerservicenummer | 000000091 |
-    En heeft 'gezag' geen derden
+      | naam                             | waarde           |
+      | type                             | GezamenlijkGezag |
+      | minderjarige.burgerservicenummer | 000000091        |
+      | ouder.burgerservicenummer        | 000000085        |
+      | derde.burgerservicenummer        | 000000061        |
```

Comment:
ik begrijp dit scenario niet
- volgens de titel is vader een niet-ouder? wat betekent vader dan? en waarom krijgt deze dan gezag?
- de titel zegt nog dat het resulteert in Voogdij. dat moet zijn gezamenlijk gezag
- Bea heeft alleen Paul als ouder, er is geen ouder 1
- er is gezag uitspraak 2D, dus Paul heeft gezag met een derde
- nergens kan je uit afleiden wie die derde is. dat moet dus een OnbekendeDerde zijn

Reply:
Je hebt gelijk. Er is verwarring ontstaan wat wij verstaan onder _niet ouder_ (derde) vanuit het domein en _niet ouder_ vanuit gezag-api.

Gezag uitspraak bevat nu `OnbekendeDerde`.